### DISCUSSION TITLE
Change led mapping

### DISCUSSION THREAD

**Discussion by rtorrente** [Koenkk/zigbee2mqtt#24834] [General]
Hello,

I just received an RGBW led strip shipped with [TS0504B](https://www.zigbee2mqtt.io/devices/TS0504B.html#tuya-ts0504b). Pairing goes well, no problem.

The problem is that the order of the strip are not RGBW but GRBW so Green and Red are inverted when i change color in controller.

Is there a way to do a remaping to set that green channel is the red channel ?

Thanks for your help,

**Comment by 53RT** [comment]
Similar problem with the [Paulmann RGB Controller](https://www.zigbee2mqtt.io/devices/500.47.html). I am not sure if it uses a third order but if set the color with
```json
{
  "state": "ON",
  "color": {
    "r": 0,
    "g": 0,
    "b": 255
  }
}
```
 I noticed the following mapping.
| Input | Output |
| -------- | ---------- |
| R | G |
| G | B |
| B | R |

So a flexible way to order the channels would be great.
